src/http2_frame_data.erl

```diff
@@ -6,12 +6,25 @@
          format/1,
          read_binary/2,
          to_frames/3,
-         to_binary/1
+         to_binary/1,
+         data/1,
+         new/1
         ]).
 
 -behaviour(http2_frame).
 
--spec format(data()) -> iodata().
+-record(data, {
+    data :: iodata()
+  }).
+-type payload() :: #data{}.
+-type frame() :: {#frame_header{}, payload()}.
```

Comment:
It was the first one I did, and I had a different idea that didn't pan out, forgot to delete.